AspNetCore.Authentication.WsFederation/WsFederationAuthenticationHandler.cs

```diff
@@ -33,7 +33,12 @@ public WsFederationAuthenticationHandler(ILogger logger)
             // Allow login to be constrained to a specific path.
             if (Options.CallbackPath.HasValue && Options.CallbackPath != Request.PathBase + Request.Path)
             {
-                return null;
+                if (Options.SkipUnrecognizedRequests)
```

Comment:
I'm thinking this should always be AuthenticationResult.Skip() since the CallbackPath doesn't match path. That automatically means this request is not meant for us.